operate/client/src/modules/utils/instance/index.ts

```diff
@@ -31,6 +33,10 @@ const getProcessName = (instance: ProcessInstanceEntity | null) => {
   return processName || bpmnProcessId || '';
 };
 
+const getProcessDefinitionName = (instance?: ProcessInstance) => {
+  return instance?.processDefinitionName || instance?.processDefinitionId || '';
+};
```

Comment:
I think it shouldn't be the responsibility of this function to guarantee if the instance exists or not

```suggestion
const getProcessDefinitionName = (instance: ProcessInstance) => {
  return instance?.processDefinitionName ?? instance.processDefinitionId;
};
```

Reply:
Agreed 👍